### ISSUE TITLE
chown /nexus-data can take a long time

### ISSUE THREAD

**Issue by bcg62** [clearent/nexus#8]
If you have large storage in nexus this chown can take a very long time. 

This also becomes an issue with livenessProbes/readinessProbes in kubernetes

Wondering if there is some way around this.

**Comment by cavemandaveman** [comment]
Yes, I've noticed this behavior too. I'm not sure of a good way around it. I am open to suggestions.

**Comment by bcg62** [comment]
https://unix.stackexchange.com/questions/245274/chown-only-where-needed-speedup-chown
